evm/gas-service.js

```diff
@@ -71,6 +71,16 @@ async function getFeeData(api, sourceChain, destinationChain) {
     return feesCache[key];
 }
 
+async function relayTransaction(chain, address, calldata, nativeValue, gasLimit) {
+    return httpPost(process.env.RELAYER_API_ENDPOINT, {
```

Comment:
avoid reading env var directly from process. This should be read via the CLI args